conan/api/subapi/cache.py

```diff
@@ -47,17 +49,21 @@ def check_integrity(self, package_list):
         checker = IntegrityChecker(app)
         checker.check(package_list)
 
-    def clean(self, package_list, source=None, build=None, download=None):
+    def clean(self, package_list, source=True, build=True, download=True, temp=True):
```

Comment:
What's the reasoning behind this specific change? This effectively makes the default API behaviour be "Clean all except if I tell you otherwise", while the CLI's is "Don't clean anything I don't tell you to", which seems weird

Reply:
👍 sounds good, thanks for the clarification!